File: Backends/C#/SuperHeroApi/SuperHeroDomain/QueryModel/ListAllHeroesPaginatedResponse.cs
using SuperHeroDomain.Infrastructure.Query;
using SuperHeroDomain.Model.HeroMaster;
using System.Collections.Generic;

namespace SuperHeroDomain.QueryModel
{
    public class ListAllHeroesPaginatedResponse : PagedServiceResponseBase
    {
        public List<Hero> Heroes { get; set; }

        public ListAllHeroesPaginatedResponse()
        {
            Heroes = new List<Hero>();
        }
    }
}

File: Backends/TypeScript/src/SuperHeroDomain/QueryModel/ListAllHeroesPaginatedResponse.ts
import { PagedServiceResponseBase } from "../Infrastructure/PagedServiceResponseBase";
import { Hero } from "../Model/HeroModel";


export class ListAllHeroesPaginatedResponse extends PagedServiceResponseBase{
    heroes: Array<Hero>;

    constructor(){
        super();
        this.heroes = new Array();
    }
}